### PR TITLE
batch sys -> job runner

### PR DESCRIPTION
**Pull request by MetRonnie**
Sibling to https://github.com/cylc/cylc-flow/pull/3992

### CODE REVIEW

File: src/tutorial/runtime/runtime-configuration.rst [hunk 1]
```diff
@@ -284,9 +286,9 @@ Start, Stop, Restart
       This sets the :term:`final cycle point` six hours after the
       :term:`initial cycle point`.
 
-      Run `cylc validate` to check for any errors::
+      Run `cylc validate` to check for any errors:
```
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:289]: The double colon is actually an RST feature, it is short-hand for `.. code-block::`:

```suggestion
      Run `cylc validate` to check for any errors::
```

File: src/tutorial/runtime/runtime-configuration.rst [hunk 2]
```diff
@@ -302,7 +304,7 @@ Start, Stop, Restart
 
       .. TODO: Add instructions for offline configuration
 
-      Generate a Datapoint API key::
+      Generate a Datapoint API key:
```
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:307]:
```suggestion
      Generate a Datapoint API key::
```

File: src/tutorial/runtime/runtime-configuration.rst [hunk 3]
```diff
@@ -411,7 +413,7 @@ Start, Stop, Restart
       .. spoiler:: Hint hint
 
          If you run ``ls work`` you should see a
-         list of cycles. Pick one of them and open the file::
+         list of cycles. Pick one of them and open the file:
```
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:416]:
```suggestion
         list of cycles. Pick one of them and open the file::
```

**Review by wxtim — APPROVED** [review]
Looks eminently reasonable, barring Oliver's comments.

**Comment by hjoliver** [comment]
Committed @oliver-sanders 's suggested changes.

**Review by hjoliver — APPROVED** [review]

**Comment by hjoliver** [comment]
Build passed.

File: src/tutorial/runtime/runtime-configuration.rst (hunk 1 above)
Comment (MetRonnie) [reply src/tutorial/runtime/runtime-configuration.rst]: Oh right. For some reason, the [rst extension for VSCode](https://github.com/vscode-restructuredtext/vscode-restructuredtext) doesn't like that; the syntax highlighting for everything after that line doesn't work

**Pull request merged**